tests/unit/test_helper.bash

```diff
@@ -406,7 +406,7 @@ server {
     proxy_set_header Connection \$http_connection;
     proxy_set_header Host \$http_host;
     proxy_set_header X-Forwarded-Proto \$scheme;
-    proxy_set_header X-Forwarded-For \$remote_addr;
+    proxy_set_header X-Forwarded-For \$proxy_add_x_forwarded_for;
```

Comment:
Should default back to `remote_addr`.

We also need a test of setting/unsetting this property to ensure it's set correctly/I dont break the option in a future refactor.